### PR TITLE
Add ImageScannerController init options

### PR DESCRIPTION
**Pull request by tomaschyly**
Our client required that the scanner starts with manual. Seeing as there are no init options, I have added them with ability to choose isAutoScanEnabled.

### CODE REVIEW

**Comment by tomaschyly** [comment]
I am not sure when I will have the time, I am quite busy. Can you look at them?

**Comment by AvdLee** [comment]
> I am not sure when I will have the time, I am quite busy. Can you look at them?

We're busy with our own roadmap. It would also be great if the one writing the implementation, also writes test as you have the best context of the implementation.

Also, if you can pull in master, that would be great! CI would run successfully again 💪

**Comment by tomaschyly** [comment]
I will try to get to it this week.

File: WeScan/ImageScannerController.swift [hunk 1]
```diff
@@ -85,6 +92,8 @@ public final class ImageScannerController: UINavigationController {
     }
 
     override public init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        self.imageScannerOptions = ImageScannerOptions()
+        
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
     }
```
Comment (wetransferplatform) [inline WeScan/ImageScannerController.swift:99]: - :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

**Pull request closed without merging**